analyses/cell-type-ewings/run-aucell-ews-signatures.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+# This script is used to get AUC values for a set of EWS-FLI1 high/low gene signatures using AUCell
+# The input gene signatures are those saved in `references/gene_signatures` and the following MSigDb gene sets: 
+
+# `STAEGE_EWING_FAMILY_TUMOR`
+# `MIYAGAWA_TARGETS_OF_EWSR1_ETS_FUSIONS_UP`
+# `MIYAGAWA_TARGETS_OF_EWSR1_ETS_FUSIONS_DN`
+# `ZHANG_TARGETS_OF_EWSR1_FLI1_FUSION`
+# `RIGGI_EWING_SARCOMA_PROGENITOR_UP`
+# `RIGGI_EWING_SARCOMA_PROGENITOR_DN`
+# `KINSEY_TARGETS_OF_EWSR1_FLII_FUSION_UP`
+# `KINSEY_TARGETS_OF_EWSR1_FLII_FUSION_DN`
```

Comment:
I think you can remove this now since we have the TSV file, but the comment above might need to be updated a bit about where the reference files are?